README.md

```diff
@@ -15,6 +15,21 @@ The library provides two main functionalities:
 The [documentation](https://craft-cli.readthedocs.io) is available in Read The Docs.
 
 
+# Setting up the environment
+
+Install at system level:
+
+    sudo snap install pyright
+
+Create a virtual environment, activate it, and install developer dependencies:
+
+    python3 -m venv env
+    source env/bin/activate
+    pip install .[dev]
+
+That's all.
+
+
```

Comment:
Just a note.

You could also setup the development environment via

```
tox --devenv <dir name>
```

But right - I just saw you install `tox` the other way round with `.[dev]` - so my suggestion does not make much sense for this setup :-)

Reply:
Yes, there are multiple ways. We just need to indicate one so we enable newcomers to start easily.